util/rowcodec/rowcodec_test.go

```diff
@@ -81,35 +75,36 @@ func (s *testSuite) TestEncodeLargeSmallReuseBug(c *C) {
 		},
 	}, nil)
 	m, err := bDecoder.DecodeToDatumMap(b, nil)
-	c.Assert(err, IsNil)
+	require.NoError(t, err)
 	v := m[smallColID]
-	c.Assert(v.GetInt64(), Equals, int64(2))
+	require.Equal(t, int64(2), v.GetInt64())
 }
 
-func (s *testSuite) TestDecodeRowWithHandle(c *C) {
+func TestDecodeRowWithHandle(t *testing.T) {
```

Comment:
You may try to run different test data with subtests.

Reply:
see also https://pkg.go.dev/testing#hdr-Subtests_and_Sub_benchmarks

Reply:
also `TestIsNormalizedDiskNameAllowed` as an example/

Reply:
done